### DISCUSSION TITLE
Add support for excel spreadsheets to application import.

### DISCUSSION THREAD

**Discussion by jortel** [konveyor/tackle2-hub#942] [Ideas]
Support Excel spreadsheet (format) in addition to CSV.

**Comment by aufi** [comment]
I'm happy to look on this task, I might have few question to clarify:
- do we want support xls and/or xlsx file formats?
- should order or columns in the spreadsheet matter? (like Tackle 2 csv import) or their title should matter
- is the expected table format be the same as in csv import?
